concepts/05 Widgets/RadioGroup/00 Overview.md

```diff
@@ -36,8 +116,103 @@ If your data is an array of objects, bind it to the **RadioGroup** using the [di
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-radio-group
+        [data-source]="dataItems"
+        [value]="dataItems[1]"
```

Comment:
Doesn't look like a good idea. It is better to bind to a component property in Angular and Vue and use the state in React.

Reply:
fixed